src/blockdev.rs

```diff
@@ -805,7 +805,7 @@ pub fn lsblk_single(dev: &Path) -> Result<HashMap<String, String>> {
 /// rereading mitigates possble issue with outdated UUIDs on different
 /// paths to the same disk: after 'ignition-ostree-firstboot-uuid'
 /// '/dev/sdaX' path gets new UUID, but '/dev/sdbX/' path has an old one
-fn lsblk_all_and_reread_partition_tables() -> Result<Vec<HashMap<String, String>>> {
+fn lsblk_all_and_reread_partition_tables(reread_pt: bool) -> Result<Vec<HashMap<String, String>>> {
```

Comment:
Since we're now not always rereading, `and_reread_partition_tables` is somewhat misleading.  Would it make sense to just call this `lsblk_all`?